frontend/src/Components/TopBarGame.jsx

```diff
@@ -37,7 +38,12 @@ const TopBarGame = ({ showTimer, showRole }) => {
                     </IconButton>
                 </div>
             )}
-            {showTimer && <div className={styles.timer}>Timer Placeholder</div>}
+            {showTimer && (
+                <div className={styles.timer}>
+                    {/* The default time just be set to 90 for now */}
+                    <Timer userPreferTime="90" />

```

Comment:
We need to come back to this and remove the hardcoding and change it to the value the server gives (in the config file)